homeassistant/components/unifi/sensor.py

```diff
@@ -339,6 +339,19 @@ class UnifiSensorEntityDescription(
         value_fn=async_device_state_value_fn,
         options=list(DEVICE_STATES.values()),
     ),
+    UnifiSensorEntityDescription[Wlans, Wlan](
```

Comment:
Is this related to my previous question, to circumvent the sensors not passing requires_admin=True?

Reply:
You need the supported_fn to make sure there is a value

Reply:
I've performed a test with a non admin account, the x_passphrase attribute is still returned but with value `None`, so it's shown `Unknown` as value in HA interface. This means it wouldn't fail, but still I've created the async_get_password_supported_fn to guarantee the `x_passphrase` is present.

P.S.: I didn't check for the value itself because a Guest WLAN might have an empty password, in this case the sensor will still be shown.

What do you think?

Reply:
Make supported_fn be x_passphrase is not None then

Reply:
> Make supported_fn be x_passphrase is not None then

Doing so will make the sensor for WLAN Guest with no password not be shown as well, even for admins, and I think they should be shown, that's my point.
I know a sensor with no value isn't important, but maybe they don't have a password now, but will have, and this sensor can be used in some automation, I don't know. Do you get my point?

Reply:
Every restart that will be reevaluated so I think it's fine

Reply:
hmm.. ok then!